.gitignore

```diff
@@ -0,0 +1,3 @@
+test.md
```

Comment:
The .gitignore is suppose to prevent the test.md file from being pushed into the repository. Kindly ensure that this is working effectively as I can see it in the repository :pray: